src/main/java/org/opensearch/geospatial/GeospatialParser.java

```diff
@@ -78,7 +78,7 @@ public static String extractValueAsString(final Map<String, Object> input, final
      * getFeatures will return features from given map input. This function abstracts the logic to parse given input and returns
      * list of Features if exists in Map format.
      * @param geoJSON given input which may contain GeoJSON Object
-     * @return List of Feature in Map
+     * @return Optional List of Feature in Map, if Feature exist
```

Comment:
I think from current description it isn't clear what we're going to return in case Feature doesn't exist.

Reply:
Sure. Will update.